Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/Concepts.java

```diff
@@ -201,35 +245,40 @@ private JsonValidationResult checkJsonObject(JSONObject jsonObject) {
                     false);
         }
 
-        if (jsonObject.get(JsonFields.CONCEPT_LIST) == null) {
+        if (jsonObject.get(JsonFields.DESCRIPTION) == null) {
             return new JsonValidationResult(
-                    "Error parsing request: please provide a concept list for the concept ('conceptlist' attribute).",
+                    "Error parsing request: please provide a description for the concept ('description' attribute).",
                     jsonObject, false);
         }
-
-        String pos = jsonObject.get(JsonFields.POS).toString();
-        if (!POS.posValues.contains(pos)) {
-            logger.error("Error creating concept from REST call. " + pos + " does not exist.");
-            return new JsonValidationResult("POS '" + pos + "' does not exist.", jsonObject, false);
-        }
-
         return new JsonValidationResult(null, jsonObject, true);
     }
 
+
     private ConceptEntry createEntry(JSONObject jsonObject, String username) {
         ConceptEntry conceptEntry = new ConceptEntry();
-        conceptEntry.setCreatorId(username);
-        conceptEntry.setSynonymIds(jsonObject.get(JsonFields.SYNONYM_IDS) != null ? jsonObject.get(
-                JsonFields.SYNONYM_IDS).toString() : "");
-        conceptEntry.setWord(jsonObject.get(JsonFields.WORD).toString());
+        if (jsonObject.get(JsonFields.WORDNET_ID) != null) {
+            conceptEntry.setWordnetId(jsonObject.get(JsonFields.WORDNET_ID).toString());
+            String[] wrappers = jsonObject.get(JsonFields.WORDNET_ID).toString().split(Constants.CONCEPT_SEPARATOR);
```

Comment:
I checked in UI. In create concept wrapper page we are able to add more than one wordnet id. And the wrapper is created for both of those. I have attached a sample screenshot for a wrapper with two wordnet ids. 
<img width="1156" alt="screen shot 2016-09-14 at 11 01 16 am" src="https://cloud.githubusercontent.com/assets/9751475/18523786/9fd3fc54-7a6a-11e6-9f88-2b67acf2e054.png">